docs/build/introduction/astar_family.md

```diff
@@ -33,40 +26,26 @@ The Shibuya native token symbol is SBY.
 
 To obtain test tokens from the faucet, please visit the Astar Portal and connect to Shibuya. If for any reason the faucet is empty, please contact the Astar team on Discord.
 
-### zKatana (Sepolia) - Deprecated
-
-An early iteration of the zkEVM testnet that used zk rollup architecture. Now deprecated.
-
-### zKyoto (Sepolia)
+### zKyoto (Sepolia) (Deprecated)
 
-zKyoto is a Validium-based network connected to the Ethereum Sepolia testnet and Polygon AggLayer. It is the testing and proving ground for dApps deployed on the Astar zkEVM.
+zKyoto was connected to the Ethereum Sepolia testnet and Polygon AggLayer. Testing ground for dApps deployed on the Astar zkEVM. This testnet has now been transitioned to [Soneium Minato](https://docs.soneium.org/).
 
 ## Mainnets
 
-Astar has three mainnets:
+Astar Collective with it's asset supports three mainnets:
```

Comment:
_Astar Collective with **its** asset supports three mainnets:_ 

or

_Astar Collective, with the ASTR token, supports three mainnets:_

Reply:
LGTM